packages/react-core/src/components/Card/examples/Card.md

```diff
@@ -828,3 +828,116 @@ class ExpandableIconCard extends React.Component {
   }
 }
 ```
+
+### Legacy selectable and selected
+
+```js
+import React from 'react';
+import {
+  Card,
+  CardHeader,
+  CardActions,
+  CardTitle,
+  CardBody,
+  Dropdown,
+  DropdownToggle,
+  DropdownItem,
+  DropdownSeparator,
+  KebabToggle
+} from '@patternfly/react-core';
+
+class SelectableCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selected: null
+    };
+    this.onKeyDown = event => {
+      if (event.target !== event.currentTarget) {
+        return;
+      }
+      if ([13, 32].includes(event.keyCode)) {
```

Comment:
Can you also add `event.preventDefault();` for this block? `Space` will cause a page down while also selecting the card currently.